.github/workflows/build-publish.yml

```diff
@@ -104,9 +104,11 @@ jobs:
     permissions:
       contents: read
       id-token: write
-    uses: smartcontractkit/.github/.github/workflows/reusable-docker-build-publish.yml@86250a243934401117c92a137579d4cd7a253572 # 2025-03-28
+    uses: smartcontractkit/.github/.github/workflows/reusable-docker-build-publish.yml@16a9ca3a2352bcb47539ebbb6f6153d657f9f8d0 # 2025-03-31
     with:
       aws-ecr-name: chainlink
+      aws-region-ecr: us-east-1
+      aws-region-gati: us-west-2
```

Comment:
For future consideration, you might be able to set all these duplicate values as [workflow wide env vars](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#defining-environment-variables-for-a-single-workflow) and then reference them through the templating language.